src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -40,46 +40,43 @@ interface FormFieldParam {
   placeholderText: string
   errorMessage: string
 }
-interface ImplicitFormFieldParam<T> {
-  value: T
+interface ImplicitParam<T, K extends keyof T> {
```

Comment:
FormField seems like an ill suited name to have here since these parameters are not in the form

Reply:
agreed